wpilibc/src/main/native/cppcs/RobotBase.cpp

```diff
@@ -35,6 +36,12 @@ int frc::RunHALInitialization() {
   }
   HAL_Report(HALUsageReporting::kResourceType_Language,
              HALUsageReporting::kLanguage_CPlusPlus, 0, GetWPILibVersion());
+
+  if (!frc::Notifier::SetHALThreadPriority(true, 40)) {
+    throw FRC_MakeError(err::Error, "{}",
+                        "Setting HAL Notifier RT priority to 40 failed\n");
+  }
```

Comment:
We should not kill the robot program due to this happening.  Print a warning, yes, but not kill the whole program.